subscription/notify-test-lambda/README.md

```diff
@@ -0,0 +1,55 @@
+This lambda is a test lambda meant to be run in SIT and WL. 
+
+A user creates an ingest granule subscription. An example follows:
+{"Name": "Ingest-Subscription-Test-Sit-http",
+ "Type": "granule",
+ "SubscriberId": "eereiter",
+ "CollectionConceptId": "C1200463968-CMR_ONLY",
+ "EndPoint": "http://internal-cmr-services-sit-internal-only-2141831226.us-east-1.elb.amazonaws.com:80/notification/tester",
+ "Mode": ["New", "Update", "Delete"],
+ "Method": "ingest",
+ "MetadataSpecification": {
+   "URL": "https://cdn.earthdata.nasa.gov/umm/subscription/v1.1.1",
+   "Name": "UMM-Sub",
+   "Version": "1.1.1"
+ }
+}
+
+Make sure the URL is the CMR internal load balancer followed by /notification/tester.
+
+Ingest a granule. 
+
+To verify that the notification is correct and that it was sent tunnel into the CMR internal load balancer
```

Comment:
... sent <<through the>> tunnel into ....?

Reply:
I changed the wording.